sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -115,51 +115,6 @@ This starter provides a convenient way to quickly access resource servers.
 2. In Controller, `@RegisteredOAuth2AuthorizedClient` can be used to  get `OAuth2AuthorizedClient`. 
    `OAuth2AuthorizedClient` contains accessToken to access related client resource.
```

Comment:
The 2 items should move to `Web application visit other resource servers`

Reply:
`or roles claim in the access token` can be deleted. Now we do not support this.

Reply:
In order to show the second feature, do we still need to use the sample code in web-api-obo, because there is no similar code in webapp
https://github.com/Azure/azure-sdk-for-java/blob/f28826e51a3f31566483756d059baa73e18a30dc/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/src/main/java/com/azure/spring/sample/aad/controller/SampleController.java#L59-L75

Reply:
> do we still need to use the sample code in web-api-obo

Just use code in webapp: ClientController


> there is no similar code in webapp

No. We have oAuth2AuthorizedClient, it can be used to access related resource server, but we didn't use it in the sample we just return the json string. It's OK to put the code in readme.

https://github.com/Azure/azure-sdk-for-java/blob/ae482017ebcbdd070eb2e6e1ad5c8d66a63da22a/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/java/com/azure/spring/sample/aad/controller/ClientController.java#L30-L36